### ISSUE TITLE
TextField with PHONENUMBER Constraint

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Perundurai-Products/microemu#5]
```
I found a strange behaviour in the TextField with PHONENUMBER constraint.
1) I cannot type the symbol '+' used in international number (like
+39xxxxx)
2) I can type the phone number with the pc keyboard but I cannot type the
number using the microemulator keyboard.
When I type a number using the microemulator keyboard the cursor
disappear.
I see the same behavior in the demo applet at
http://www.microemu.org/demo.html in TextField section.
Try to insert a phone number in the field 'Phone number' using the
microemulator keyboard.
So I think it is a bug!

Regards,
Ugo Chirico
http://www.cryptware.it
http://www.ugosweb.com
```

Original issue reported on code.google.com by `bar...@gmail.com` on 18 Nov 2009 at 11:08

**Comment by GoogleCodeExporter** [comment]
Original comment by `bar...@gmail.com` on 8 Jan 2010 at 11:12
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in trunk (future 3.0 version)
```

Original comment by `bar...@gmail.com` on 14 Jan 2010 at 1:32
- Changed state: **Fixed**
- Added labels: **Milestone-3.0**
